snykTask/src/index.ts

```diff
@@ -205,7 +206,8 @@ async function runSnykTest(
     !fs.existsSync(jsonReportOutputPath) &&
     snykTestExitCode === CLI_EXIT_CODE_SUCCESS
   ) {
-    const echoToolRunner = tl.tool('echo');
+    // pipe console json output to snykToHtml and file for subsequent conversion again
+    const echoToolRunner = tl.tool(snykToHtmlPath);
```

Comment:
yes, `snyk-to-html` [accepts piped input](https://docs.snyk.io/snyk-cli/scan-and-maintain-projects-using-the-cli/cli-tools/snyk-to-html#snyk-code-command) from `--json` output.  `.pipeExecOutputToTool` takes a 1st argument that accepts a piped input. The 2nd argument also writes that piped output to file which is used for the html conversion and display.

Reply:
issue: understood, but a test is still needed to validate the change in implementation integrates as expected. i.e. no errors, no changes in behaviour (to avoid major breaking changes), etc.

Reply:
Pls see a new `execute pipeExecOutputToTool to snyk-to-html` test-case that sends a piped input to `snyk-to-html` execution and also input written to a file location.